cabinet/vault.py

```diff
@@ -62,7 +68,49 @@ def add(self, item):
         fname = os.path.join(data_path, fname)
         self._file_write(fname, content)
 
-    def get(self, name):
+    def update(self, name, new_item):
```

Comment:
Here we should start adding some docstrings, I know that it's not time to prettify everything, but it would help if we add docstrings to the new functions, and it will reduce the future work. Also, it'd help me to better understand what is the intention here. As an example, why you `del new_item['content']`?, the client that uses this library may don't expect this.